### ISSUE TITLE
Quick fix group of three issues - update form elements

### ISSUE THREAD

**Issue by DanielCouzens** [epimorphics/ppd-explorer#217]
form updates please see cells

- [x] Missing fieldset https://docs.google.com/spreadsheets/d/1YS3N5ps9pJGFOxBDi9C2DwA37WQPIVax/edit?gid=154688571#gid=154688571&range=A6:K6
- [x] Incorrectly associated form label https://docs.google.com/spreadsheets/d/1YS3N5ps9pJGFOxBDi9C2DwA37WQPIVax/edit?gid=154688571#gid=154688571&range=A7:J7
- [x] input box border contrast https://docs.google.com/spreadsheets/d/1YS3N5ps9pJGFOxBDi9C2DwA37WQPIVax/edit?gid=154688571#gid=154688571&range=A8:J8

**Comment by DanielCouzens** [comment]
in review see PR https://github.com/epimorphics/ppd-explorer/pull/222

**Comment by DanielCouzens** [comment]
@tomguilbert this is ready to check on the dev instance https://hmlr-dev-pres.epimorphics.net/app/ppd/

This is difficult to check as two changes are invisible to the user. 
If you want to see a more obvious difference use the silktide extension in chrome. Choose accessibility.

There are still accessibility errors with this update. but you should see significantly less compared to production.

**Comment by tomguilbert** [comment]
Thanks @DanielCouzens - these all look good now.
